docs/reference/presigned-url.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: default
+title: Presigned URL
+description: Configuring lakeFS to use presigned URLs.
+parent: Reference
+nav_order: 100
+has_children: false
+---
+
+# Presigned URL
+{: .no_toc }
+
+{% include toc.html %}
+
+lakeFS provides the option to access the data directly from the storage and not through lakeFS using presinged URL.
+lakeFS can generate a presigned URL to the object store by a user who has access to the object. The presigned URL can be used to access the object.
+lakeFS assume by default the object storage support presinged URLs, if not it should be disabled in the configuration.
+
+# Using presigned URLs in the UI
+
+For the UI to fetch objects using a presigned URL to the object storage (instead through lakeFS), need to enable the presigned URL support UI in the lakeFS configuration and add CORS (Cross-Origin Resource Sharing) permissions to the bucket.
```

Comment:
You must enable the presigned URL support UI (name the configuration used to enable it) in the lakeFS configuration (link the text to our configuration reference) and add CORS (Cross-Origin Resource Sharing) permissions to the bucket for the UI to fetch objects using a presigned URL (instead of through lakeFS).

Reply:
done